docs/usage/configuration-options.md

```diff
@@ -1663,10 +1663,37 @@ See also `matchPackagePrefixes`.
 
 The above will match all package names starting with `eslint` but exclude ones starting with `eslint-foo`.
 
+### matchCategories
+
+Use `matchCategories` to restrict rules to a particular language or group.
+The categories can be found in the [manager documentation](./modules/manager.md).
+
+<!-- prettier-ignore -->
+!!! warning
+Rules with `matchCategories` are only applied _after_ extraction of a repository.
+You can not use `matchCategories` to stop processing a repository.
+You can still use `matchCategories` to stop or change updating the dependencies though.
```

Comment:
```suggestion
    Rules with `matchCategories` are only applied _after_ extraction of a repository.
    You can not use `matchCategories` to stop processing a repository.
    You can still use `matchCategories` to stop or change updating the dependencies though.
```

We should use four spaces here. The easy way to remember this is that the text should match the key-word for the admonition.

Or maybe Prettier messed up the formatting automatically... 😄

Reply:
🤔  looks like my IDE removed it during cut and copy and I haven't realised it. 😄